website/thaliawebsite/templates/base.html

```diff
@@ -212,8 +218,6 @@
                 src="{% static "js/jquery.min.js" %}"></script>
         <script type="text/javascript"
                 src="{% static "js/js.cookie.min.js" %}"></script>
-        <script type="text/javascript"
-                src="{% static "js/bootstrap.bundle.min.js" %}"></script>
```

Comment:
I don't understand why we no longer need javascript

Reply:
we include bootstrapjs already on top

Reply:
Oh I see it now. From the package. Does it have an async attribute? Otherwise it should probably be [best](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup) at the bottom of the page.

Reply:
Adding to this: I noticed that this means we would start using the jdelivr cdn. I personally like keeping our used bootstrap version in the codebase. Even tho that means it needs manual updates.